README.md

```diff
@@ -0,0 +1,19 @@
+-Create a bot on Discord
```

Comment:
I think github needs spaces after each of these dashes for markdown to work right

Reply:
updated